front-end/src/app/components/manage/manage.component.spec.ts

```diff
@@ -19,20 +22,28 @@ describe('ManageComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it("should create", () => {
     expect(component).toBeTruthy();
   });
 
-  it('should render header', () => {
+  it("should render header", () => {
     const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.querySelector('h3').textContent).toContain('Acties');
+    expect(compiled.querySelector("h3").textContent).toContain("Acties");
   });
 
-  it('should have all buttons present', () => {
+  it("should have all buttons present", () => {
     const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.getElementsByClassName('start').item(0).textContent).toContain('Start');
-    expect(compiled.getElementsByClassName('test').item(0).textContent).toContain('Test');
-    expect(compiled.getElementsByClassName('stop').item(0).textContent).toContain('Stop');
-    expect(compiled.getElementsByClassName('reset').item(0).textContent).toContain('Reset');
+    expect(
+      compiled.getElementsByClassName("start").item(0).textContent
+    ).toContain("Start");
+    expect(
+      compiled.getElementsByClassName("test").item(0).textContent
```

Comment:
Can you add a test for the test behavior?

Reply:
Done, good suggestion